docs/machine-learning/tutorials/image-classification-api-transfer-learning.md

```diff
@@ -0,0 +1,529 @@
+
+    ```csharp
+    var trainingPipeline = mlContext.Model.ImageClassification(
+            featuresColumnName: "Image",
+            labelColumnName: "LabelAsKey",
+            arch: ImageClassificationEstimator.Architecture.ResnetV2101,
+            epoch: 100,
+            batchSize: 20,
+            testOnTrainSet: false,
+            metricsCallback: (metrics) => Console.WriteLine(metrics),
+            validationSet: validationSet,
+            reuseTrainSetBottleneckCachedValues: true,
+            reuseValidationSetBottleneckCachedValues: true,
+            disableEarlyStopping:false)
+        .Append(mlContext.Transforms.Conversion.MapKeyToValue("PredictedLabel"));
+    ```
+
+    The `ImageClassification` estimator takes in several parameters:
+
+    - `featuresColumnName` is the column that is used as input for the model.
+    - `labelColumnName` is the column for the value to predict.
+    - `arch` defines which of the pretrained model architectures to use. This tutorial uses the 101-layer variant of the ResNetv2 model.
+    - `epoch` specifies the maximum number of iterations over the entire dataset throughout the training process. The higher the number, the longer the model trains for and potentially the better model that is produced.
+    - `batchSize` is the number of samples to use at a time for training. During one epoch, multiple batches equal to the batchSize are used to train and update the model. The lower the number, the less memory required when each batch is processed.
+    - `testOnTrainSet` tells the model to measure performance against the training set when no validation set is present.
+    - `metricsCallback` binds a function to track the progress during training.
+    - `validationSet` is the [`IDataView`](xref:Microsoft.ML.IDataView) containing the validation data.
+    - `reuseTrainSetBottleneckCachedValues` tells the model whether to use the cached values from the bottleneck phase in subsequent runs. The bottleneck phase is a one-time pass-through computation that is computationally intensive the first time it is performed. If the training data does not change and you want to experiment using a different number of epochs or batch size, using the cached values significantly reduces the amount of time required to train a model.
+    - `reuseValidationSetBottleneckCachedValues` is similar to `reuseTrainSetBottleneckCachedValues` only that in this case it's for the validation dataset.
+    - `disableEarlyStopping` tells the Image Classification whether to employ an early stopping strategy. As the model searches for the optimal values that will help it make accurate predictions during training, performance may increase or decrease. Ultimately, if the model reaches the last epoch, it may be the case that the patterns it learned from training are suboptimal. Early stopping monitors training for these drops in performance and stops the training process in an effort to preserve an optimal version of the model.
```

Comment:
```suggestion
    - `disableEarlyStopping` tells the ImageClassification estimator whether to employ an early stopping strategy. As the model searches for the optimal values that will help it make accurate predictions during training, performance may increase or decrease. Ultimately, if the model reaches the last epoch, it may be the case that the patterns it learned from training are suboptimal. Early stopping monitors training for these drops in performance and stops the training process in an effort to preserve an optimal version of the model.
```

Reply:
Instead of "the Image Classification" did you mean "the ImageClassification estimator" that this list is about?